### PR TITLE
Veggie burger button z-index

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Fix `Veggie Burger` z-index when showing the menu on mobile

### Before
Screenshot 2021-04-16 at 17 02 58

### After
Screenshot 2021-04-16 at 16 58 20

## Why?
Fix order of layout

### CODE REVIEW

File: src/web/lib/getZIndex.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const indices = [
 	'banner',
 	'dropdown',
 	'burger',
-	'stickyNav',
+	'expanded-veggie-menu',
```
Comment (liywjl) [inline src/web/lib/getZIndex.tsx:30]: `expanded-veggie-menu` makes more sense as it is used in both, stick and classic Nav

**Review by shtukas — APPROVED** [review]

File: src/web/components/Nav/StickNavTest/ExpandedMenu/VeggieBurgerMenu.tsx [hunk 2]
```diff
@@ -13,62 +13,59 @@ const screenReadable = css`
...
+const beforeAfterStyles = css`
+	content: '';
+	background-color: currentColor;
+`;
+const lineStyles = css`
+	height: 2px;
+	left: 0;
+	position: absolute;
+	width: 20px;
+`;
```
Comment (liywjl) [inline src/web/components/Nav/StickNavTest/ExpandedMenu/VeggieBurgerMenu.tsx:25]: Just moving out `const`s from `veggieBurgerIconStyles`

File: src/web/components/Nav/Nav.tsx [hunk 3]
```diff
@@ -90,7 +90,7 @@ export const Nav = ({ format, nav, subscribeUrl, edition }: Props) => {
                         var showMoreButton = document.getElementById('${showMoreButtonId}')
                         var veggieBurger = document.getElementById('${veggieBurgerId}')
                         var expandedMenuClickableTags = document.querySelectorAll('.selectableMenuItem')
-                        var expandedMenu = document.getElementById('expanded-menu')
+                        var expandedMenu = document.getElementById('expanded-menu-root')
```
Comment (liywjl) [inline src/web/components/Nav/Nav.tsx:93]: 2 `div`s had `expanded-menu` as an ID. I separated the parent `div` to have suffix `-root`

File: src/web/components/Nav/ExpandedMenu/VeggieBurgerMenu.tsx [hunk 4]
```diff
@@ -13,62 +13,60 @@ const screenReadable = css`
...
 
-	return css`
-		background-color: currentColor;
-		/*
+const lineStyles = css`
+	height: 2px;
+	left: 0;
+	position: absolute;
+	width: 20px;
+`;
```
Comment (liywjl) [inline src/web/components/Nav/ExpandedMenu/VeggieBurgerMenu.tsx:26]: lifting these `const`s out of `veggieBurgerIconStyles`

File: src/web/components/Nav/ExpandedMenu/ExpandedMenu.tsx [hunk 5]
```diff
@@ -95,7 +97,7 @@ export const ExpandedMenu: React.FC<{
 	nav: NavType;
 }> = ({ display, nav }) => {
 	return (
-		<div id="expanded-menu">
+		<div id="expanded-menu-root">
```
Comment (liywjl) [inline src/web/components/Nav/ExpandedMenu/ExpandedMenu.tsx:100]: `expanded-menu` ID was already being used

Comment (oliverlloyd) [reply src/web/components/Nav/ExpandedMenu/ExpandedMenu.tsx:100]: Ah, this was being flagged up by accessibility checkers - I've been wanting to fix this for ages!

**Review by oliverlloyd — APPROVED** [review]
Great work!

The comments really help with reviewing and it's great to see that double `id` be fixed

**Pull request merged**